registry/remote/example_test.go

```diff
@@ -451,6 +479,51 @@ func ExampleRepository_Fetch_artifactReferenceManifest() {
 	// {"mediaType":"application/vnd.cncf.oras.artifact.manifest.v1+json","artifactType":"example/signature","blobs":null,"subject":{"mediaType":"application/vnd.oci.image.manifest.v1+json","digest":"sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b","size":24}}
 }
 
+// ExampleRepository_Fetch_blobsOfArtifactManifest gives an example of pulling the blobs
+// of an artifact manifest.
+func ExampleRepository_Fetch_blobsOfArtifactManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+
+	// Fetch the artifact manifest by digest. The digest can be obtained from Referrers API.
+	exampleDigest := "sha256:73bccdadf23b5df306bf77b23e1b00944c7bbce44cf63439afe15507a73413b5"
+	_, rc, err := repo.FetchReference(ctx, exampleDigest)
+	if err != nil {
+		panic(err)
+	}
+	defer rc.Close()
+
+	// Parse the pulled manifest and fetch its blobs.
+	pulledContent, err := io.ReadAll(rc)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(string(pulledContent))
+
+	var pulledManifest artifactspec.Manifest
+	if err := json.Unmarshal(pulledContent, &pulledManifest); err != nil {
+		panic(err)
+	}
+	for _, blob := range pulledManifest.Blobs {
+		content, err := content.FetchAll(ctx, repo, ocispec.Descriptor{
```

Comment:
Nice-to-have: since user might reuse this code snippet to download really large blobs, ever consider using concurrent programming fo the blob fetching? 

If you do that, `Unordered output:` comment primitive need to be used, for more information please refer to https://pkg.go.dev/testing#hdr-Examples

Reply:
Let's keep it simple in the examples.